templates/base/api/types.ts.hbs

```diff
@@ -551,3 +551,6 @@ function parseArrayDiff<T>(buf: _Reader, innerParse: () => T): (T | typeof _NO_D
   }
   return arr;
 }
+
+export type StateId = string;
+export type AppId = string;
```

Comment:
we shouldn't export `AppId` since it's not a concept that users need to know about